brainstormforce/ultimate-addons-for-gutenberg
# Fix: Timeline - Icon miss-aligned issue.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/0276c160-546a-484c-8c29-78f93f038788?redirect=true&view=06c01065-22db-4b44-9a83-f0eceb06e66a&t=attributes

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop timeline blocks (content/post)
- Check the center layout editor and front.
- Check timeline Icon is centered align or not.
- Check by changing Orientation and Arrow Alignment. 

### Checklist:
